usefulUtilities/webAppTemplate/dbMaker.js

```diff
@@ -0,0 +1,59 @@
+const Database = require('./dbClass.js');
+const dbInfo = require("./dbInfo.json");
+const baseConfig = {
+    host: dbInfo.config.mySQLHost,
+    user: dbInfo.config.mySQLUsername,
+    password: dbInfo.config.mySQLPassword,
+};
+
+// Grab the tables from the config file and dynamically make the query
+let convertTables = (tables) => {
```

Comment:
why is this called "convert" and not "create" ?

Reply:
oh because it was converting the tables object from the config into the full SQL query.  change to create?

Reply:
yeah, i think so - the final verb of the function itself is to create, not to convert